web/app/templates/account/login.html

```diff
@@ -8,102 +8,103 @@
 
 {% block content %}
 {% get_providers as socialaccount_providers %}
+<div class="container">
```

Comment:
This <div> can be moved into `account/base.html` to further DRY the code. But it's not a big deal.

Reply:
I didn't find a way to do it without wrapper_container

Reply:
This is an example of how you can implement it.
![Screen Shot 2022-02-26 at 1 25 57 PM](https://user-images.githubusercontent.com/779786/155859601-2de5b25d-89ef-455e-b389-8f3210b1db4f.png)

 But now I'm not convinced myself that this is a better code. Just wanted to show you a way that can be done. Please merge without making such change.